html-forms/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="get" >
+      <label for="email">Email</label>
+      <div>
+        <input type="text" name="email" id="email">
+      </div>
+      <label for="coding options">Why is coding cool?</label>
+      <div>
+        <select name="coding options" id="coding options">
+          <option value="select">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+          </div>
+        </select>
+        <p>Favorite Language (choose one)</p>
+        <div>
+          <input type="radio" name="language" id="language" value="HTML"><label for="html">HTML</label>
+          <input type="radio" name="language" id="language" value="CSS"><label for="label">CSS</label>
+          <input type="radio" name="language" id="language" value="JavaScript"><label for="JavaScript">JavaScript</label>
```

Comment:
The `id` for each of these elements is the same, which is not allowed in HTML. The value for the `id` attribute on the `input` should match the value of the `for` attribute on its corresponding `label`